### ISSUE TITLE
New Library resource - Cloud Library ADE books will not De-DRM

### ISSUE THREAD

**Issue by dwall115** [apprenticeharper/DeDRM_tools#1803]
I downloaded a few new books to import into my Calibre Library and was surprised when latest Calibre and 7.2.1 wouldn't remove DRM.  (It's been working fine since April release).  So I went back to Calibre 4.2.3 and DeDRM 6.8.1 without success.  Do you know if something has changed in the Adobe DRM schema with this new resource.  I'll be happy to provide any log files or other information that will diagnose the issue.  thank you in advance for all the work you do, and the excellent tool you've provided.
d

**Comment by dwall115** [comment]
update with log file.

I've installed ADE 2.0.1, Calibre 4.2.1 and DeDRM 6.8.0.  ADE is setup with my authorization (e-mail/password), and downloads books just fine.  I can read them in Adobe DE.  As I surmised however, there is no default key in Adobe DE, as this log file indicates.  If there is a way to create a default key (I have Python and PyCrypto installed), perhaps that may do the trick?

LOG FILE FOLLOWS;

calibre 4.21 [64bit]  embedded-python: True is64bit: True
Windows-10-10.0.19041-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19041')
Python 2.7.16+
Windows: ('10', '10.0.19041', 'SP0', u'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: DeDRM (6, 8, 0)
calibre 4.21 [64bit]  embedded-python: True is64bit: True
Windows-10-10.0.19041-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19041')
Python 2.7.16+
Windows: ('10', '10.0.19041', 'SP0', u'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: DeDRM (6, 8, 0)
Turning on automatic hidpi scaling
devicePixelRatio: 1.0
logicalDpi: 96.0 x 96.0
physicalDpi: 81.5518394649 x 81.6428571429
Using calibre Qt style: True
[0.00] Starting up...
[0.00] Showing splash screen...
[0.36] splash screen shown
[0.36] Initializing db...
[0.36] db initialized
[0.36] Constructing main UI...
[1.28] main UI initialized...
[1.28] Hiding splash screen
[18.70] splash screen hidden
[18.70] Started up in 18.70 seconds with 79 books
stdout+stderr from file dialog helper: ['', '']
piped data from file dialog helper: ['\x05\xae\xaff\xb5\xdaA\xe3\xd8\x99 \xc6Q\x0bKT\x15\xf8\xa7\x9dt\x92\xc7\x82\xce\xf1\x92\xd6\x05\xe93G', 'C:\\Users\\dwall\\Documents\\My Digital Editions\\Marauder.epub']
DeDRM v6.8.0: Trying to decrypt Marauder.epub
DeDRM v6.8.0: Verifying zip archive integrity
DeDRM v6.8.0: Marauder.epub is a secure Adobe Adept ePub
DeDRM v6.8.0: Looking for new default Adobe Digital Editions Keys after 0.1 seconds
DeDRM v6.8.0: Exception when getting default Adobe Key after 0.1 seconds
Traceback (most recent call last):
  File "calibre_plugins.dedrm.__init__", line 351, in ePubDecrypt
  File "calibre_plugins.dedrm.adobekey", line 379, in adeptkeys
  File "calibre_plugins.dedrm.adobekey", line 359, in CryptUnprotectData
ADEPTError: Failed to decrypt user key key (sic)
DeDRM v6.8.0: Ultimately failed to decrypt after 0.1 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "site-packages\calibre\customize\ui.py", line 172, in _run_filetype_plugins
  File "calibre_plugins.dedrm.__init__", line 637, in run
  File "calibre_plugins.dedrm.__init__", line 408, in ePubDecrypt
DeDRMError: DeDRM v6.8.0: Ultimately failed to decrypt after 0.1 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Added Marauder to db in: 0.1
Added 1 books in 0.7 seconds
Exception TypeError: "'NoneType' object is not callable" in <bound method NativeFunction.__del__ of <calibre_plugins.dedrm.adobekey.NativeFunction object at 0x00000156D10BEE48>> ignore

Any direction would be appreciated.  Thank you for your time.
d
